docs/assets/archi.blockdiag.txt

```diff
@@ -0,0 +1,34 @@
+blockdiag {
```

Comment:
next step: generate this file dynamically ? :p

Reply:
is there a reasonable reason to do it for a template that won't change so much ?

Reply:
I believe this was a joke.